trillian_log_api.proto

```diff
@@ -154,6 +154,19 @@ message GetLeavesByIndexResponse {
     repeated LogLeaf leaves = 2;
 }
 
+message GetLeavesByRangeRequest {
+  int64 log_id = 1;
+  int64 start_index = 2;
+  int64 count = 3;
+}
+
+message GetLeavesByRangeResponse {
+  // Returned log leaves starting from the start_index of the request, in order.
+  // There may be fewer than request.count leaves returned, if the requested
+  // range extended beyond the size of the tree.
```

Comment:
Or if the server opted to return fewer than reqeuested...

Reply:
Done.